core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -559,15 +558,6 @@ static SqlOperatorTable operatorTableFor(SqlLibrary library) {
   }
 
   @Test void testArithmeticOperatorsFails() {
-    expr("^power(2,'abc')^")
```

Comment:
why delete these tests? What happens to them? They look like good tests.

Reply:
The `POWER` function is no longer available in `SqlStdOperatorTable`. The implementation to use depends on which library is active since PostgreSQL can use a different return type.

This test fails as it was since `POWER` is not in `StdOperatorTable`. Is there a way to specify the library to use for the test?

Reply:
Yes, see the operatorTableFor() function.

Reply:
Updated.